contributor/es/01-introduction-article-es.asciidoc

```diff
@@ -0,0 +1,63 @@
+Esta persona puede, o no, ser parte o verse como parte de la comunidad.
+Sin embargo, para muchas personas hay una especie de camino que los colaboradores pueden hacer, desde conocer la comunidad hasta usar el producto de la comunidad para interactuar con los miembros de la comunidad, y finalmente comenzar a contribuir.
+Por último, algunos colaboradores pueden convertirse en https://innersourcecommons.org/learn/learning-path/trusted-committer[Trusted Committers].
+
+=== Relación con otros roles
+
+Como Colaborador en una comunidad de InnerSource, interactuará con otras personas que desempeñan otros roles de InnerSource, como https://innersourcecommons.org/learn/learning-path/trusted-committer[_Trusted Committer_] o https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] y posiblemente con otros colaboradores.
+A veces, estos roles pueden ser desempeñados por la misma persona, como Trusted Committer y Product Owner en pequeños proyectos.
+
+Esta sección le brinda una muy breve descripción general de los otros dos roles, pero nos gustaría animarle a que lea el https://innersourcecommons.org/learn/learning-path/trusted-committer[introductory article of the Trusted Committer role], y le recomendamos que lea también el artículo https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[Lowering the barriers to entry] antes de profundizar en los detalles del rol de Colaborador en esta sección.
+También puede ver los videos (https://innersourcecommons.org/learn/learning-path/trusted-committer[introduction], https://innersourcecommons.org/learn/learning-path/trusted-committer/05/[lowering the barriers to entry]) en lugar de leer los artículos.
+
+==== Trusted Committer
+
+Un Trusted Committer será su anfitrión durante su estancia en la comunidad anfitrión.
+Son los responsables del repositorio de código del proyecto y acercarán su contribución a la producción una vez sea aceptada.
+Su función es guiarle para contribuir a su comunidad. Ellos pueden ayudarlo directamente o proporcionarle información que le permita continuar por sí mismo. Esta información podría ser, reglas internas establecidas para revisiones, plantillas de propuestas para cambios más importantes, referencias a la documentación o secciones de código relevantes para su contribución.
+
+También deben preocuparse por la calidad del producto, la sostenibilidad y la evolución del proyecto desde una perspectiva técnica y general, por reducir la barrera para hacer contribuciones para todos, así como por el cuidado de su comunidad en general.
+Cuidar de la comunidad implica mantenerla saludable, elevarla a ella y a sus participantes, y defender sus necesidades en su organización.
+
+==== Product Owner
+
+El rol de https://innersourcecommons.org/learn/learning-path/product-owner[_Product Owner_] tiene cierta similitud con el rol de product owner de su proyecto.
```

Comment:
I propose "el rol habitual del product owner en su proyecto". It deviates a bit from the original phrasing as it assigns "average" to "role" instead of "project" but I think it sticks better to the actual meaning.

Reply:
👍